### PR TITLE
Log repr of incoming message only in debug mode

### LINKED ISSUE #74: Unpreventable call to `response.__repr__` in `NSQConnection._parse.data`
Here https://github.com/list-family/ansq/blob/master/ansq/tcp/connection.py#L405 in the following call to logger it will always call `str(message)`.

``` 
self.logger.debug("NSQ: Got data: %s", truncate_text(str(response)))
```

This forces the call to `response.__repr__` and consumes a lot of resources on large amounts of data and could not be prevented.

@atugushev suggests to make this logger call conditional basing on `ConnectionOptions.debug` to prevent excessive computation of `response.__repr__`.

Alternative is getting rid of `truncate_text` and `str` calls altogether. This will prevent calling `response.__repr__` in case of debug logging level is disabled but could introduce much more data being logged in case of debug level being enabled. It seems like this could actually be useful for someone willing to debug the data being received.

### PR DESCRIPTION
**Pull request by atugushev**
Resolved #74.

### CODE REVIEW

File: ansq/utils.py [hunk 1]
```diff
@@ -179,9 +180,14 @@ def get_logger(
...
+) -> typing.Union[bytes, str]:
     """Truncate a given `text` if the `limit` is reached"""
     if limit <= 0:
         raise ValueError("limit must be greater than 0")
 
-    return text[:limit] + "..." if len(text) > limit else text
+    if len(text) <= limit:
+        return text
+
+    return text[:limit] + b"..." if isinstance(text, bytes) else text[:limit] + "..."
```
Comment (lig) [inline ansq/utils.py:193]:
```suggestion
    return text[:limit] + (b"..." if isinstance(text, bytes) else "...")
```

Comment (lig) [reply ansq/utils.py:193]: BTW, it looks like we can safely let go the `str` type here. I'd like to see the library operating binary data only.

**Review by lig — COMMENTED** [review]
Added some comments. Overall I'm ok with the PR as it is now.

File: ansq/utils.py (hunk 1 above)
Comment (atugushev) [reply ansq/utils.py]: >  I'd like to see the library operating binary data only.

Good suggestion! Addressed in b36b63d

**Review by lig — APPROVED** [review]
👍🏻

**Comment by atugushev** [comment]
Thanks @lig!

**Pull request merged**